src/java/bobbybot/Deadline.java

```diff
@@ -0,0 +1,52 @@
+package bobbybot;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/** Represents a bobbybot. Task that should be completed by a specified Date and Time
+ *  @author mokdarren
+ */
+public class Deadline extends Task{
```

Comment:
There seems to be a missing whitespace after Task. There are multiple occurrences of this throughout, so I'll leave it to you to find them :D